client.go

```diff
@@ -278,14 +279,14 @@ func (c *Client) SetBasicAuth(username, password string) *Client {
 	return c
 }
 
-// SetAuthToken method sets bearer auth token header in the HTTP request. For Example:
-// 		Authorization: Bearer <auth-token-value-comes-here>
+// SetAuthToken method sets the auth token header (default scheme: Bearer) in the HTTP request. For Example:
```

Comment:
Wordsmithing...

> The SetAuthToken method sets the authentication token of the Authorization header for all HTTP requests.  The default authentication scheme is Bearer, but this can be changed with SetAuthScheme.

Reply:
SetAuthToken method sets the auth token of the `Authorization` header for all HTTP requests. The default auth scheme is `Bearer`, it can be customized with the method `SetAuthScheme`.